README.md

```diff
@@ -156,6 +156,22 @@ and add this config to your `package.json`:
 
 See https://github.com/okonet/lint-staged#configuration for more details about how you can configure 🚫💩 lint-staged.
 
+Alternately you can just save this script as `pre-commit` in your `.git/hooks` directory and give it execute permissions:
```

Comment:
I would word it

> save this script as `.git/hooks/pre-commit` and give it execute permission:

otherwise it's not 100% clear how the file is named and where it lives